src/components/Button.js

```diff
@@ -10,27 +10,35 @@ const ButtonComponent = ({onPress, title}) => {
     };
     const buttonStyle = {
         ...styles.button,
-        backgroundColor: isPressed ? 'red' : 'green',
```

Comment:
this is an if case, if you make the options same it serves no purpose. just write `'red'` here if that will be the only option

Reply:
Fixed.